catalog/transformer/catalog-transformer-streaming-impl/src/test/java/org/codice/ddf/transformer/xml/streaming/impl/TestXmlInputTransformer.java

```diff
@@ -157,12 +157,11 @@ public void testGml3Conversion() throws FileNotFoundException, CatalogTransforme
     xmlInputTransformer.setSaxEventHandlerConfiguration(Collections.singletonList("gml-handler"));
     GmlHandlerFactory factory = new GmlHandlerFactory();
     factory.setGml3ToWkt(gml3ToWkt);
-    xmlInputTransformer.setSaxEventHandlerFactories(
-        Collections.singletonList((SaxEventHandlerFactory) factory));
+    xmlInputTransformer.setSaxEventHandlerFactories(Collections.singletonList(factory));
     Metacard metacard = xmlInputTransformer.transform(inputStream);
     assertThat(
         metacard.getAttribute(Metacard.GEOGRAPHY).getValue(),
-        is("POLYGON ((35 10, 10 20, 15 40, 45 45, 35 10), (20 30, 35 35, 30 20, 20 30))"));
```

Comment:
DDF requries WKT to be Lon/Lat and the Gml3ToWkt convert didn't account for this.